### ISSUE TITLE
Cannot run with --height larger than display height

### ISSUE THREAD

**Issue by gncgnc** [patriciogonzalezvivo/glslViewer#137]
My monitor is 1366x768 and I'm on Ubuntu 18.04 . When I try to run glslViewer with a height larger than the display height, the window is clipped to what fits on the screen. This occurs even when running in headless mode, and screenshots have their bottoms clipped off like so 
 
![try](https://user-images.githubusercontent.com/18458167/60753638-8423b700-9fde-11e9-971a-51f2a0a12f81.png)
(this is supposed to be a 1000x1000 image)

I was getting expected behaviour before I updated my Nvidia driver. I may have changed some other things as well, but unfortunately I cannot remember what else may be causing this. (I also installed openFrameworks which may be messing with the glfw version perhaps?) 

I could just stick to low resolutions, but high resolution output would be very nice to have, also it was working before so I wonder what may be causing the problem.

**Comment by patriciogonzalezvivo** [comment]
You didn't recompile glslViewer? just update the Nvidia Drivers?

**Comment by gncgnc** [comment]
No, I did not recompile glslViewer, although I did recompile glfw in case that was the issue, but no luck. 

Do you recommend removing and reinstalling glslViewer? Will it detect the new drivers and build accordingly?

**Comment by gncgnc** [comment]
I removed and reinstalled glslViewer with no effect. 

Additional info: I can manually resize the glslViewer window to my liking (1000x1000) after it starts running, but the startup height is clipped to 740. I can also get 1000x1000 screenshots this way, but the issue still persists in headless mode.

**Comment by patriciogonzalezvivo** [comment]
If the problem start with the new Nvidia drivers is probably coming from there. Still similar issue manifest since MacOS Mojave

**Comment by patriciogonzalezvivo** [comment]
Duplicate https://github.com/patriciogonzalezvivo/glslViewer/issues/125
